### ISSUE TITLE
Coordinates bugged on some older maps

### ISSUE THREAD

**Issue by Dovvah** [Marlamin/wow.tools#55]
For example on Eastern Kingdoms 5.4.7.18019 shows 85_83 tile for middle of EK

Same thing happens for Kalimdor, 8.1.5.28938 

Doesn't happen for every map of the same build number; coordinates are fine on Azeroth 4.0.1.11927 but bugged on Kalimdor 4.0.1.11927

**Comment by Marlamin** [comment]
Indeed. Nice catch! Will hopefully have some time to look at this before Shadowlands hits. If not, it might be a while.

**Comment by Marlamin** [comment]
Applied a temporary fix for this in 1e80c41, it now shows a message instead of faulty coordinates. Will have to add coordinate support for these maps at some point down the road, so leaving this open for then.

**Comment by Dovvah** [comment]
Dunno if this is related and you probably already know but the ADT grid is wrong on these unsupported maps which is probably the cause. 

It seems like it's the exact same offset for every map, exactly one ADT up and to the left 

http://puu.sh/FK97b/d72fef9494.jpg 
http://puu.sh/FK9bR/9f87411c1f.jpg 
http://puu.sh/FK9ct/3673c7cef9.jpg
http://puu.sh/FK9fd/1ea4b1aba8.jpg
